ai_edge_torch/generative/quantize/README.md

```diff
@@ -18,7 +18,29 @@ Once converted, you will get a quantized `.tflite` model which will be ready for
 
 In the current release, the following schemes are supported:
 
-* Dynamic range quantization with FP32 activations and INT8 weights for linear ops
-* FP16 quantization with FP16 weights and FP32 activations and computation for all ops
+* Dynamic range quantization: FP32 activations, INT8 weights, and INT8 computation
```

Comment:
Re: `INT8 computation`, I have been wondering for some time about using a more precise term here. The alternatives that appeal to me are:

1.  "integer accumulation"

2. "int8 operands"

Either one of those would work I think. And similar for `FP32 computation` -> "floating point accumulation"/"fp32 operands". Maybe it's a small difference, but I think it does provide additional clarity on what is meant by "computation" here. What do you think?

Reply:
I've been interpreting "computation" as math operations or "accumulation and operands", so INT8 computation probably doesn't make sense and at the minimum we should change it to integer computation and floating point computation. "Accumulation" feels a bit too specific to certain ops IMO. WDYT of just "integer computation" or "floating point computation"?

Reply:
yes "integer" (or "float") "computation" for now sounds good.